[2024-05-03T14:01:45.636+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-03T14:01:45.718+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.upload_file_items manual__2024-05-03T13:39:42.184579+00:00 [queued]>
[2024-05-03T14:01:45.757+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.upload_file_items manual__2024-05-03T13:39:42.184579+00:00 [queued]>
[2024-05-03T14:01:45.757+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-05-03T14:01:45.868+0000] {taskinstance.py:2327} INFO - Executing <Task(LocalFilesystemToGCSOperator): upload_file_items> on 2024-05-03 13:39:42.184579+00:00
[2024-05-03T14:01:45.967+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=21799) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-03T14:01:45.978+0000] {standard_task_runner.py:63} INFO - Started process 21907 to run task
[2024-05-03T14:01:45.974+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'upload_file_items', 'manual__2024-05-03T13:39:42.184579+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp8xvlon55']
[2024-05-03T14:01:45.980+0000] {standard_task_runner.py:91} INFO - Job 111: Subtask upload_file_items
[2024-05-03T14:01:46.307+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.upload_file_items manual__2024-05-03T13:39:42.184579+00:00 [running]> on host 0db60b1edb75
[2024-05-03T14:01:48.427+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='upload_file_items' AIRFLOW_CTX_EXECUTION_DATE='2024-05-03T13:39:42.184579+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-03T13:39:42.184579+00:00'
[2024-05-03T14:01:48.432+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-03T14:01:48.665+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-05-03T14:01:48.676+0000] {base.py:84} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-05-03T14:01:50.304+0000] {gcs.py:557} INFO - File /opt/***/dags/data/olist_order_items_dataset.csv uploaded to olist_order_items_dataset.csv in storage-capstone bucket
[2024-05-03T14:01:50.313+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-03T14:01:50.349+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=upload_file_items, execution_date=20240503T133942, start_date=20240503T140145, end_date=20240503T140150
[2024-05-03T14:01:51.475+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-03T14:01:52.836+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-03T14:01:52.851+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
